index.html

```diff
@@ -0,0 +1,179 @@
+          our invitation list including organisations and individuals.
+        </p>
+
+        <h3>2023.10.18(THU) ~ 2023.10.18(FRI)</h3>
+        <p id="venue">@ Madrid,Spain</p>
+      </section>
+    </div>
+    <section class="program" id="program">
+      <div class="programtitle">
+        <h3>Main Program</h3>
+        <div class="liner"></div>
+      </div>
+      <div class="program-container">
+        <div class="lecture">
+          <div class="lec-image"> <img src="images/speak.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Lecture</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image"><img src="images/exhibit.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Exhibition</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image"> <img src="images/forum.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Forum</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image">
+            <img src="images/wkshop.png" alt="Program" />
+          </div>
+          <div class="lec-h3">
+            <h3>Workshop</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image"><img src="images/ignite.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Algorithms</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+      </div>
+      <button type="button" id="join">Join Conference</button>
+
+      <div class="program-title">
+        <h3>Featured Developers</h3>
+        <div class="liner"></div>
+      </div>
+
+
+
+      <section class="speakers" id="speakers">
+
+      </section>
+
+
+
+    </section>
+  </main>
+  <footer>
+
+    <div class="patners" id="patners">
+      <h3>PARTNERS</h3>
+      <div class="liners"></div>
+    </div>
+    <div class="logo1">
+      <img src="images/open.png" alt="openAI">
+      <img src="images/alpha.png" alt="Alphabet">
+      <img src="images/taiwan.png" alt="Taiwan">
+      <img src="images/tsla.png" alt="Tesla">
+
```

Comment:
- [ ] Great job so far, for this section in the design the line after the header partner should be centre according to the header please fix that, and also the icons/images should be aligned centre horizontally and vertically so that they are all in the same position, kindly fix that.
- [ ] [ _OPTIONAL_]  It's also best to use the color specified in the design which is **#272a31**

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/163636181-426feae7-a734-4d6e-aeb3-5beb2eccb0f0.png)

### Behace Design
![image](https://user-images.githubusercontent.com/59416136/163636221-1aac920c-e615-4ee4-9509-85c4074fb12a.png)